### ISSUE TITLE
[BUG] The onnxsim command cannot be used after pip installation of v0.4.0

### ISSUE THREAD

**Issue by PINTO0309** [daquexian/onnx-simplifier#195]
1. **Describe the bug**
    The `onnxsim` command cannot be used after pip installation of v0.4.0.
    - Install
      ```bash
      $ pip3 install -U pip && pip3 install onnxsim --upgrade
    
      $ onnxsim -h
      onnxsim: command not found
      ```

2. **My workaround**
    I have confirmed that building from source code and installing works fine. :+1: 
    ```bash
    $ git clone https://github.com/daquexian/onnx-simplifier.git && cd onnx-simplifier
    $ git checkout d98687251e232f56f884c3a6e36167300f6ac9e0
    
    $ git submodule update --init --recursive
    $ pip install -e .
    
    $ onnxsim yolov7_HxW.onnx yolov7_HxW.onnx
    Simplifying...
    Finish! Here is the difference:
    ┏━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
    ┃            ┃ Original Model ┃ Simplified Model ┃
    ┡━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
    │ Add        │ 3              │ 3                │
    │ Concat     │ 27             │ 24               │
    │ Constant   │ 29             │ 0                │
    │ Conv       │ 92             │ 92               │
    │ Gather     │ 9              │ 6                │
    │ MaxPool    │ 8              │ 8                │
    │ Mul        │ 101            │ 101              │
    │ Pow        │ 3              │ 3                │
    │ Reshape    │ 6              │ 6                │
    │ Resize     │ 2              │ 2                │
    │ Shape      │ 9              │ 6                │
    │ Sigmoid    │ 92             │ 92               │
    │ Split      │ 3              │ 3                │
    │ Transpose  │ 3              │ 3                │
    │ Unsqueeze  │ 12             │ 6                │
    │ Model Size │ 141.7MiB       │ 141.7MiB         │
    └────────────┴────────────────┴──────────────────┘

    ```

**Comment by daquexian** [comment]
Thanks for your report! I'll check it soon.

**Comment by daquexian** [comment]
Could you please try `python3 -m onnxsim -h`?

**Comment by PINTO0309** [comment]
Yes. That command did not work correctly either.

**Comment by daquexian** [comment]
I cannot reproduce it on my side. A possibly related context is that in v0.4.0 both the `onnxsim` and `python3 -m onnxsim` are provided by a "internal" package `onnxsim-no-ort`,  `pip install -e .` also installs this internal package. And this internal package is a dependency of package `onnxsim`.

Could you please create a clean virtual environment, run `pip install onnxsim` and check whether package `onnxsim-no-ort` is installed?

**Comment by daquexian** [comment]
BTW, in next version `onnxsim-no-ort` will only contain c++ compiled libraries and the python part like `onnxsim` and `python3 -m onnxsim` command will be provided by package `onnxsim`. Hopefully it will fix your problem.

**Comment by PINTO0309** [comment]
It seems to be a problem specific to my environment. Quite a few dependencies have been broken, which may interfere with the operation of other packages, but for now it is not a problem. :+1: 

I will try to use onnxsim on Docker for a while. :smile_cat: 

```bash
$ pip install onnxsim onnxsim-no-ort --force-reinstall

Successfully installed commonmark-0.9.1 numpy-1.23.0 onnx-1.12.0 
onnxsim-no-ort-0.4.0 protobuf-3.20.1 pygments-2.12.0 rich-12.4.4 typing-extensions-4.3.0

$ onnxsim -h
usage: onnxsim [-h] [--enable-fuse-bn] [--skip-fuse-bn] [--skip-optimization [SKIP_OPTIMIZATION [SKIP_OPTIMIZATION ...]]] [--skip-constant-folding]
               [--input-shape INPUT_SHAPE [INPUT_SHAPE ...]] [--overwrite-input-shape OVERWRITE_INPUT_SHAPE [OVERWRITE_INPUT_SHAPE ...]]
               [--test-input-shape TEST_INPUT_SHAPE [TEST_INPUT_SHAPE ...]] [--skip-optimizer SKIP_OPTIMIZER [SKIP_OPTIMIZER ...]] [--skip-shape-inference]
               [--dynamic-input-shape] [--input-data-path INPUT_DATA_PATH [INPUT_DATA_PATH ...]] [--custom-lib CUSTOM_LIB] [--include-subgraph]
               [--unused-output UNUSED_OUTPUT [UNUSED_OUTPUT ...]] [--no-large-tensor]
               input_model output_model [check_n]

positional arguments:
  input_model           Input ONNX model
  output_model          Output ONNX model
  check_n               Check whether the output is correct with n random inputs

$ pip list | grep onnxsim
onnxsim                           0.4.0
onnxsim-no-ort                    0.4.0 
```

I will close the issue.

**Comment by daquexian** [comment]
Glad to see the problem got solved :)

**Comment by daquexian** [comment]
> @daquexian Problem is not resolved.
> 
> ![image](https://user-images.githubusercontent.com/31848222/178742184-21310509-c24c-43c6-9c7a-7e4bddb5e88f.png)

What's your python version and os? Thanks

**Comment by chooga** [comment]
I'm having the same Problem on Ubuntu 18.04 and Python 3.6.

**Comment by daquexian** [comment]
@dexception @chooga I published v0.4.1 to fix this python 3.6 problem. Please have a try.
